cmd/network-passt-binding/domain/configurator.go

```diff
@@ -103,7 +104,9 @@ func lookupIfaceByAliasName(ifaces []domainschema.Interface, name string) (int,
 }
 
 func (s PasstNetworkConfigurator) generateInterface() (*domainschema.Interface, error) {
-	// TODO: return error in case iface model is virtio but virtio network is not possible
+	if s.options.VirtioNetProhibited && (s.vmiSpecIface.Model == vmschema.VirtIO || s.vmiSpecIface.Model == "") {
+		return nil, fmt.Errorf("in-kernel virtio-net device emulation '/dev/vhost-net' not present")
+	}
```

Comment:
Passt seem to work with non passt model types e.g.:e1000, I think we can simplify it and remove this checking.